CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ cmake_minimum_required(VERSION 3.4)
 
 include(ExternalProject)
 
-project( CCTag )
+project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
```

Comment:
With this you can do the other way around, cmake can generate the version file with

```c++
[version.hpp]
#define CCTAG_VERSION_MAJOR @PROJECT_VERSION_MAJOR@
//etc
```
that can be included by the headers of the lib or by 3rd party code (the same way i did the config.hpp)